.gitignore

```diff
@@ -37,6 +37,7 @@
 /bazel-genfiles
 /bazel-out
 /bazel-testlogs
+user.bazelrc
```

Comment:
Is this a drive-by?

Reply:
This is a recommended pattern and was used in testing this Pr. I'd recommend it stay if possible